libraries/resource_providers/api200/logical_interconnect_provider.rb

```diff
@@ -129,6 +149,10 @@ def update_ethernet_settings
       end
 
       def update_port_monitor
+        @item.retrieve! || raise("#{@resource_name} '#{@name}' not found!")
+        ports = convert_keys(@new_resource.port_monitor, :to_s)
+        load_port(ports) if ports['enablePortMonitor']
+        @item['portMonitor'] = ports
```

Comment:
Why not include this into `load_port` ?
Also, what if the user specifies a `data['portMonitor']` ? It is currently being overwritten by even `{}`, which is the default for `port_monitor`, right?

Reply:
I understood. I'll change to prevent this from happening.